<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRelationCG</class>
 <widget class="QDialog" name="EditRelationCG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Relationship</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/genius-icon.png</normaloff>
    <normalon>:/icons/images/genius-icon.png</normalon>images/genius-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditRelationCG{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(185, 185, 185, 255), stop:1 rgba(225, 225, 225, 255));
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>260</width>
     <height>566</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="selectedComputerListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Linked Geniuses&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="allGeniusComboBox">
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="pushButtonRemoveGenius">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButtonRemoveGenius {
	padding: 6px 15px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(181, 181, 181, 255), stop:1 rgba(255, 255, 255, 255));
	border: 2px solid;
	border-color: transparent;
	border-radius: 16px;
}

#pushButtonRemoveGenius::enabled {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(117, 34, 34, 255), stop:1 rgba(255, 48, 48, 255));
	color: white;
}

#pushButtonRemoveComputer::enabled::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(74, 21, 21, 255), stop:1 rgba(255, 48, 48, 255));
}</string>
        </property>
        <property name="text">
         <string>Remove selected genius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="linkedGeniusesListWidget"/>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pushButtonSaveRelationCG">
      <property name="styleSheet">
       <string notr="true">#pushButtonSaveRelationCG{
	color: rgb(255, 255, 255);
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.0149254 rgba(31, 107, 31, 255), stop:1 rgba(110, 189, 100, 255));
	padding: 6px 15px;
	border: 2px solid;
	border-color: transparent;
	border-radius: 16px;
}

#pushButtonSaveRelationCG::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.0149254 rgba(10, 35, 10, 255), stop:1 rgba(110, 189, 100, 255));
}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
